File: src/app/models/asset-addition.model.ts
import { NgbDateStruct } from "@ng-bootstrap/ng-bootstrap";

export class AssetAddition{
    id:number;
    assetCatagoryId:number=0;
    assetID:number=0;
    model:string;
    serial:string;
    purchesPrice:number;
    confiruration:string;
    warrentyType:string;
    purchesate:string;
    purchesDateNg:NgbDateStruct;
    warrentydate:string;
    warrentyDateNg:NgbDateStruct;
    note:string;
    isActive:number;
    companyId:number;
}

File: WebAPI/HRMS/Models/Property/AssetAdditionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace HRMS.Models
{
    public class AssetAdditionModel
    {
        public int ID { get; set; }
        public int AssetCatagoryId { get; set; }
        public string AssetCategoryName { get; set; }
        public int AssetID { get; set; }
        public string Model { get; set; }
        public string Serial { get; set; }
        public decimal PurchesPrice { get; set; }
        public string Confiruration { get; set; }
        public string WarrentyType { get; set; }
        public DateTime Purchesate { get; set; }
        public DateTime Warrentydate { get; set; }
        public string Note { get; set; }
        public int Isactive { get; set; }
        public int CompanyID { get; set; }
    }
}